doc/test_concept.md

```diff
@@ -0,0 +1,31 @@
+# PSENscan test concept
+
+![test architecture](test_architecture.svg)
+
+## ROS-dependent code
+
+### ROS parameter handler
```

Comment:
we need a section about acceptance tests

Reply:
Done